app/Models/Context.php

```diff
@@ -216,27 +217,29 @@ public static function getRevertState($state) {
 	public static function currentGet($array = false) {
 		if (self::$current_get['all']) {
 			return 'a';
-		} elseif (self::$current_get['starred']) {
+		}
+		if (self::$current_get['starred']) {
```

Comment:
Fine for this time, but such examples of minor restructuring are in my opinion more negative than positive, as they pollute changelogs without clear added value (making it more instance more difficult to find regressions, etc.), make reviews longer, incur a risk of introducing bugs, etc.
And by the way, on a personal note, I find the previous syntax both easier to read and more robust: for instance if one of the cases does not return anymore, you might end up entering another case without warning, while with the previous syntax the error would be caught by PHPStan. So the new syntax has removed a security barrier.